### ISSUE TITLE
Stops after 30 min

### ISSUE THREAD

**Issue by Yalmarr** [finndev/PokeBuddy#820]
Hi guys,

My bot always stops after +- 30 mins.. at first i thought it might be ip ban but it is still showing pokestops on the map, but the bot just freezes.. anyone else with the same problem ?
I can still change settings and all but the bot is just not doing anything after 30 min
Maybe because its a beta version but shouldn't everyone have that problem then ?

**Comment by osiriselite** [comment]
Im in the same boat i just reset vpn and it starts working half of the time the other half i just restart the app

**Comment by XeShaw** [comment]
Yeah, I have this problem too. Not sure what is causing it as most people don't anymore from what I have read.
